packages/rum-core/src/domain/configuration/configuration.spec.ts

```diff
@@ -453,7 +458,7 @@ describe('serializeRumConfiguration', () => {
         ? `use_${CamelToSnakeCase<Key>}`
         : Key extends 'trackLongTasks'
           ? 'track_long_task' // oops
-          : Key extends 'applicationId' | 'subdomain'
+          : Key extends 'applicationId' | 'subdomain' | 'remoteConfigurationId'
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Don't we want to track `remoteConfigurationId` in our telemetry? If it's for privacy reason, maybe we can at least have a flag like `useRemoteConfig: boolean`?